rs/tests/networking/BUILD.bazel

```
@@ -88,6 +112,33 @@ system_test_nns(
         "//rs/rust_canisters/canister_test",
         "//rs/test_utilities",
         "//rs/test_utilities/types",
+        "//rs/tests/networking:canister_http_correctness_tests_lib",
```

nit: I guess we can use relative paths here? 
```suggestion
        ":canister_http_correctness_tests_lib",
```